CHANGELOG.md

```diff
@@ -41,6 +41,17 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - HCL (a.k.a Terraform) experimental support
 
 ### Changed
+- **METRICS COLLECTION CHANGES**: In order to target development of Semgrep features, performance improvements,
+  and language support, we have changed how metrics are collected by default
+  - Metrics collection is now controlled with the `--metrics` option, with possible values: `auto`, `on`, or `off`
+  - `auto` will send metrics only when rules are pulled from the Semgrep Registry, but not send
```

Comment:
Not clear to me that when a run has rules from both Registry and a local file, metrics will be sent for both